bundle/resolvers/resolvers.go

```diff
@@ -0,0 +1,253 @@
+		w := b.WorkspaceClient()
+		entity, err := w.RegisteredModels.GetByName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.FullName), nil
+	}
+	resolvers["repos"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Repos.GetByPath(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.Id), nil
+	}
+	resolvers["schemas"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Schemas.GetByName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.FullName), nil
+	}
+	resolvers["service-principals"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.ServicePrincipals.GetByDisplayName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.Id), nil
+	}
+	resolvers["tables"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Tables.GetByName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.TableId), nil
+	}
+	resolvers["token-management"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.TokenManagement.GetByComment(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.TokenId), nil
+	}
+	resolvers["tokens"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Tokens.GetByComment(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.TokenId), nil
+	}
+	resolvers["users"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Users.GetByUserName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.Id), nil
+	}
+	resolvers["volumes"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Volumes.GetByName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.VolumeId), nil
+	}
+	resolvers["warehouses"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Warehouses.GetByName(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.Id), nil
+	}
+	resolvers["workspace"] = func(ctx context.Context, b *bundle.Bundle, name string) (string, error) {
+		w := b.WorkspaceClient()
+		entity, err := w.Workspace.GetByPath(ctx, name)
+		if err != nil {
+			return "", err
+		}
+
+		return fmt.Sprint(entity.ObjectId), nil
+	}
```

Comment:
Not all of these (majority?) are not relevant in the context of DABs. I would prefer an allow-list to ensure we only expose lookups that make sense. For ex, clusters, instance pools, warehouses, are all great. But the workspace lookup is (I believe) a lookup by path. All the UC related ones as well, they are already named so don't need a lookup. The tokens and token management ones are not relevant.